TESTS/network/emac/emac_test_initialize.cpp

```diff
@@ -36,20 +48,66 @@
 #endif
 
 #include "emac_api.h"
+
+#endif
+
+#include "emac_initialize.h"
 #include "emac_tests.h"
 #include "emac_util.h"
 
 using namespace utest::v1;
 
 static unsigned char eth_mac_addr[ETH_MAC_ADDR_LEN];
 
+#if !MBED_CONF_APP_TEST_EMAC
 static char emac_if_link_state_change_cb_data[] = "link_state_change_cb_data";
 static char emac_if_link_input_cb_data[] = "link_input_cb_data";
-
-static bool emac_if_init(void);
+#endif
 
 void test_emac_initialize()
 {
+    worker_loop_init();
+
+#if MBED_CONF_APP_TEST_EMAC
+
+#if MBED_CONF_APP_TEST_ETHERNET
+
+    static EthernetInterface *network_interface = new EthernetInterface;
+
+#elif MBED_CONF_APP_TEST_WIFI
+
+    // Add wifi classes here (e.g. OdinWiFiInterface and RTWInterface)
+    static WiFiInterface *network_interface = new WiFiInterface;
```

Comment:
I think we this should be the else after the Odin and Realtek cases just to make those work as-is for now.

Cleverer stuff like `WiFiInterface::get_default_instance()` or `NetworkInterface::get_default_instance()` to come later.

Reply:
Added Odin and Realtek interfaces.